jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/PythonExtensions.kt

```diff
@@ -3,6 +3,8 @@
 
 package software.aws.toolkits.jetbrains.services.lambda
 
+import com.intellij.execution.configurations.GeneralCommandLine
```

Comment:
At what point do we think these `{Language}Extensions.kt` files are getting too big and should actually be broken up into a new namespace and separate source files?

(hint: I think the time is now 😺 )

Reply:
Oh, I thought that long ago with Java one 😛,

I'll make a follow up PR to refactor python and Java